app.js

```diff
@@ -156,12 +156,14 @@ var ontorrent = function (torrent) {
           type: 'list',
           name: 'file',
           message: 'Choose one file',
-          choices: engine.files.map(function (file, i) {
-            return {
-              name: file.name + ' : ' + bytes(file.length),
-              value: i
-            }
-          })
+          choices: Array.from(engine.files)
+            .sort((file1, file2) => file1.path.localeCompare(file2.path))
+            .map(function (file, i) {
+              return {
+                name: file.name + ' : ' + bytes(file.length),
+                value: engine.files.map(file=>file.path).indexOf(file.path)
```

Comment:
This is inefficient. You recreate the same map of filenames on every iteration.

Also, there's a codestyle issue somewhere that fails tests.